SCClassLibrary/Common/Control/Server.sc

```diff
@@ -897,6 +973,16 @@ Server {
 		this.initTree;
 	}
 
+	freeMyGroup {
```

Comment:
`freeMyDefaultGroup`? or just `freeDefaultGroup`, since we don't seem to use "my" in other code

Reply:
I find freeMyDefaultGroup clearer for humans, so changed it to that.